Client/Ecosia/UI/NTP/AboutEcosia/NTPAboutEcosiaCellViewModel.swift

```diff
@@ -0,0 +1,86 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Core
+
+class NTPAboutEcosiaCellViewModel {
+    var sections = AboutEcosiaSection.allCases
+    weak var delegate: SharedHomepageCellDelegate?
+    var expandedIndex: IndexPath?
+    
+    func deselectExpanded() {
+        guard let index = expandedIndex else { return }
+        expandedIndex = nil
+        delegate?.invalidateLayout(at: [index])
+    }
+}
+
+extension NTPAboutEcosiaCellViewModel: HomepageViewModelProtocol {
+    var isEnabled: Bool {
+        User.shared.showAboutEcosia
+    }
+    
+    var sectionType: HomepageSectionType {
+        .aboutEcosia
+    }
+    
+    var headerViewModel: LabelButtonHeaderViewModel {
+        .init(title: .localized(.aboutEcosia),
+              isButtonHidden: true)
+    }
+    
+    func section(for traitCollection: UITraitCollection) -> NSCollectionLayoutSection {
+        let height = NTPAboutEcosiaCell.UX.height
+        let count = CGFloat(numberOfItemsInSection())
+        let item = NSCollectionLayoutItem(
+            layoutSize: .init(widthDimension: .fractionalWidth(1),
+                              heightDimension: .estimated(height))
+        )
+        let group = NSCollectionLayoutGroup.horizontal(
+            layoutSize: .init(widthDimension: .fractionalWidth(1),
+                              heightDimension: .estimated(height*count)),
+            subitem: item,
+            count: 1
+        )
+        let section = NSCollectionLayoutSection(group: group)
+        section.contentInsets = sectionType.sectionInsets(traitCollection)
+        section.boundarySupplementaryItems = [
+            .init(layoutSize: .init(widthDimension: .fractionalWidth(1),
+                                    heightDimension: .estimated(100)),
+                  elementKind: UICollectionView.elementKindSectionHeader,
+                  alignment: .top)
+        ]
+        return section
+    }
+    
+    func numberOfItemsInSection() -> Int {
+        sections.count
+    }
+}
+
+extension NTPAboutEcosiaCellViewModel: HomepageSectionHandler {
+    
+    func configure(_ cell: UICollectionViewCell, at indexPath: IndexPath) -> UICollectionViewCell {
+        guard let cell = cell as? NTPAboutEcosiaCell else {
+            return UICollectionViewCell()
+        }
```

Comment:
it's one of these things were there's no real right or wrong as the case should never happen. Personally I prefer `guard let` instead of `force unwrap`